webview-ui/test-generation/src/state.ts

```diff
@@ -1,13 +1,13 @@
 import { v4 as uuidv4 } from 'uuid';
-import { Credentials, TestRecord, Vote } from '../../../src/types';
+import { Credentials, TestRecord, Vote, AppInfo } from '../../../src/types';
 
 export interface Assertion {
   value: string;
   key: string;
 }
 
 export interface Platform {
-  name: 'iOS' | 'Android';
+  name: 'ios' | 'android' | 'iOS' | 'Android';
```

Comment:
I think this may be hiding some relevant historic information.

Am I correct to assume that the previously stored records are using `iOS` and `Android`?

If I was someone new to code, and I see this new definition now, I don't know _why_ we have the case differences. If you intend to keep these changes, it'd be helpful to the future-us to have an inline explanation 🙏 

💡 I think there's a more important gotcha here though that introduces some future pitfalls though and worth a re-evaluation. You can see it in https://github.com/saucelabs/vscode-scriptiq/blob/ff56521e78cdac110a94513251c93745d59c45d0/webview-ui/test-generation/src/state.ts#L399-L399 for instance, forcing you to potentially lie to the compiler.

Food for thought:
1. Do we need to change _our_ platform definition just because App Storage defines it differently? Or should we transform what's coming from App Storage to our case standard? This change would be more isolated and require no backend change on our side either.
2. Should we change our definition to lower case? In which case the correct way would be to do a full data migration (storage.ts [has a placeholder exactly for this use case](https://github.com/saucelabs/vscode-scriptiq/blob/ff56521e78cdac110a94513251c93745d59c45d0/src/storage.ts#L47-L47)). Is it a annoying? Yes, but that's the consequence of not massaging App Storage responses to our needs.
3. We're in beta and can use it to our advantage! Let's say, much like above, that maybe we do want to lowercase our platform definition _eventually_. We can transform the data coming from App Storage to our needs _now_ and schedule the changes for a lowercase switch for a `v1.0.0` release, not needing the data migration change.

Reply:
I will update this PR to modify the API data to be upper case and add a new ticket to update our platform definitions to match the API and be lower case, as both are acceptable inputs as platform names into appium and that will allow for greater consistency.